### PR TITLE
Integration with Travis CI

### LINKED ISSUE #9: Integration with Travis CI
The tools come with PHPUnit-based tests. In particular, unit and static tests. In order to ensure quality of incoming pull requests, it would be beneficial to implement integration with the Travis CI, similarly to how it's done for [Magento 2 CE repository](https://github.com/magento/magento2).

AC:
- Unit tests run upon every push to the repo
- Static tests run upon every push to the repo
- Badge displaying a state of the latest build on a `develop` branch

### PR DESCRIPTION
**Pull request by sshymko**
- Implemented separate builds for unit and static tests
- Implemented the build status badge for the `develop` branch
- Removed declaration of compatibility with PHP 7 incompatible with other dependencies
- Resolved #9 Integration with Travis CI

### CODE REVIEW

**Comment by sshymko** [comment]
Builds on Travis CI [run successfully](https://travis-ci.org/sshymko/code-migration/branches).

**Comment by sshymko** [comment]
**Acceptance instructions:**
1. Activate builds for repository https://travis-ci.org/magento/code-migration/ by the account owner
2. Merge the current pull request to trigger execution of new builds

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         }
     },
     "require": {
-        "php": "~5.5.0|~5.6.0|~7.0.0",
+        "php": "~5.5.0|~5.6.0",
```
Comment (sshymko) [inline composer.json]: Some dependencies of the tool are incompatible with PHP 7, so that Composer will abort the installation.

**Comment by tang-yu** [comment]
Thanks!

**Pull request merged**